src/bundle/Resources/public/js/scripts/core/custom.dropdown.js

```diff
@@ -142,6 +142,13 @@
             }
 
             if (this.itemsContainer.closest(SELECTOR_MODAL)) {
+                const heightModal = this.itemsContainer.closest('.modal-content').getBoundingClientRect().height;
+                const leftToBottom = heightModal - itemsContainerTop;
```

Comment:
```suggestion
                const modalHeight = this.itemsContainer.closest('.modal-content').getBoundingClientRect().height;
                const leftToBottom = modalHeight - itemsContainerTop;
```

Reply:
Also I'm not perfectly sure about `leftToBottom` variable, it sounds quite vague for me

Reply:
@GrabowskiM maybe heightToBottom?